examples/secure-env-vars/main.tf

```diff
@@ -1,12 +1,16 @@
 locals {
-  project_id            = "<your-project-id>"
-  network               = "<your-network>"
-  subnetwork            = "<your-subnetwork>"
-  region                = "<your-region>"
-  zone                  = "<your-zone>"
-  image                 = "<your-image>"
-  domain                = "<example.com>"
-  managed_zone          = "<your-managed-zone>"
+  project_id                   = "<your-project-id>"
+  network                      = "<your-network>"
+  subnetwork                   = "<your-subnetwork>"
+  region                       = "<your-region>"
+  zone                         = "<your-zone>"
+  image                        = "<your-image>"
+  domain                       = "<example.com>"
+  managed_zone                 = "<your-managed-zone>"
+  enable_oslogin               = false
```

Comment:
Can we remove the locals from all the examples, as these are optional variables anyways and now add extra entries to the list in the example.

Reply:
the main reason I created this PR was to allow to make these settings configurable ... i.e. logging is being transitioned from old to new ( using now fluentbit etc ). The main idea was to make sure we have terraform clearly state the requirements and do not depend on defaults as well for different versions of COS etc.